dell-dss-9500M/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Dell DSS 9500M",
+  "rules": [
+    {
+      "path": "ipmi-fru.Builtin FRU Device (ID 0).Board Mfg",
+      "contains": "DELL"
+    },
+    {
+      "path": "ipmi-fru.Builtin FRU Device (ID 0).Board Part Number",
+      "contains": "0G50D4A00"
```

Comment:
@BillyAbildgaard and @pengz1 , confirming this is a released product commercially available.  Support is being generated as part of RI-138 and RI-139 for an engagement to demo RackHD+ Shovel integration with Openstack.